baseline_capture.sh

```
@@ -0,0 +1,68 @@
+#     JAR=path/to/daikonplusplus.jar   # optional override
+#     DP_OPENAI_MODEL=gpt-4o-mini      # recorded in manifest
+#
+# After running, commit the baseline directory to git:
+#     git add testdata/daikonpp_runs/baseline
+#
+# Later regression tests re-run the full pipeline and compare all
+# generated artifacts to this baseline to ensure that future changes
+# (in code or model) do not alter deterministic outputs.
+# ----------------------------------------------------------------------
+
+#!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)